doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1640,18 +1669,46 @@ The view is applied to the top assumption.
   variable, using the |SSR| ``case`` tactic described in
   :ref:`the_defective_tactics_ssr`. It then behaves as the corresponding
   branching :token:`i_pattern`, executing the
-  sequence:token:`i_item`:math:`_i`  in the i-th subgoal generated by the
+  sequence :token:`i_item`:math:`_i`  in the i-th subgoal generated by the
   case analysis; unless we have the trivial destructing :token:`i_pattern`
   ``[]``, the latter should generate exactly m subgoals, i.e., the top
   variable should have an inductive type with exactly m constructors [#7]_.
   While it is good style to use the :token:`i_item` i * to pop the variables
   and assumptions corresponding to each constructor, this is not enforced by
   |SSR|.
+``(`` :token:`i_item` * ``| … |`` :token:`i_item` * ``)``
+  *branching* :token:`i_pattern`. With no exception
+  it executes the sequence :token:`i_item`:math:`_i` on the i-th subgoal.
+  Raises an error if the number of goals and the number of :token:`i_item`
+  mismatch. Note that ``()`` can be used to assert that there is only one
+  goal.
+``[^`` :token:`ident` ``]``
```

Comment:
Same thing, this is not an `i_pattern` but an `i_block`. Either split the explanations according to the grammar or refactor the grammar and the description completely (I wouldn't mind the latter).